core/schema/modulesource.go

```diff
@@ -1936,6 +1938,95 @@ func (s *moduleSourceSchema) moduleSourceGeneratedContextDirectory(
 		}
 	}
 
+	// Generate clients
```

Comment:
This entire function definitely needs to be split down if that's at all possible - it's getting *very* long.